### ISSUE TITLE
Qustion about Zero-shot evaluation on UCIT

### ISSUE THREAD

**Issue by ZDYoung0519** [Ghy0501/MCITlib#4]
I found several questions when I tried to evaluate the LLaVA-1.5 (pre-training stage) on your proposed benchmark.

(1) ImageNet-R: The accuracy on ImageNet-R with your proposed Insturction file is
```
Samples: 3000
Accuracy: 32.10
```
The accuracy is higher than the reported results in your papaer, which may indicate that the pre-training stage of LLaVA also overlaps  with ImageNet-R?

(2) ArxivQA: I have triey to generate answers with LLaVA pretrain:
```
{"question_id": "5309", 
"question": "What can be inferred about the error bars associated with the \u0393\u2080 measurements?They represent the confidence interval for each measurement.\nThey decrease as the momentum p increases.\nThey are smallest at the highest and lowest values of momentum p.\nThey are consistent across all values of momentum p.\nRationale:The error bars for \u0393\u2080, represented by the vertical lines extending from the data points on the lower plot, appear to be of consistent length across different values of momentum p. This suggests that the uncertainty or variability associated with the measurements of \u0393\u2080 is roughly the same throughout the range of momentum p presented.\nAnswer with the option's letter from the given choices directly.", 
"answer": "D", 
"prediction": "The error bars for \u0393\u2080, represented by the vertical lines extending from the data points on the lower plot, appear to be of consistent length across different values of momentum p. This suggests that the uncertainty or variability associated with the measurements of \u0393\u2080 is roughly the same throughout the range of momentum p presented."}
```
```
{"question_id": "1046", 
"question": "Which dataset shows a clearer deviation from the power-law distribution in the tail of the distribution?A. Sogou-08\nB. Sogou-11\nC. Yahoo-10\nD. All datasets show no deviation\nRationale:In subfigure (a), the Sogou-08 dataset shows a deviation from the straight line in the tail on the log-log plot, indicating a deviation from the power-law distribution.\nAnswer with the option's letter from the given choices directly.", 
"answer": "A", 
"prediction": "The answer is D. All datasets show no deviation."}
```

Some questions seem to lack of option's letters (case1). And for case 2, I dont't know how to evaluate this task propoerly, as no eval scripts for zero-shot is provided. 

I have noticed that in the appendix of your paper:
```
Specifically, for tasks that answer a single option or word, we use pred.upper() in ground_truth.upper() to determine whether the answer is correct.
```
Does this apply to ArxivQA as well? If so, such an assessment also seems problematic. For example in case 2, answer `A` is in the prediciton, but the prediciton is actually wrong.

**Comment by Ghy0501** [comment]
Regarding question 1, we did not rely solely on zero-shot performance to determine whether a dataset exhibited information leakage, since large models like LLaVA naturally possess some degree of generalization. Instead, our focus was on whether there was a substantial gap between fine-tuned and zero-shot performance to assess the dataset’s utility. In fact, most datasets—even those with low zero-shot performance—were deemed unsuitable if fine-tuning failed to yield significant improvement.

Regarding question 2, we will carefully review the data you provided and make updates promptly. Thank you for your valuable insight.
